packages/genesys-spark-components/src/components/beta/gux-selector-cards/gux-selector-card/gux-selector-card.tsx

```diff
@@ -52,6 +62,18 @@ export class GuxSelectorCard {
     }
   }
 
+  @Listen('keydown')
+  onKeydown(event: KeyboardEvent): void {
+    switch (event.key) {
+      case 'ArrowRight':
+      case 'ArrowDown':
+      case 'ArrowLeft':
+      case 'ArrowUp':
```

Comment:
I don't see why this is needed. Does the native browser not handle this for you?